app.js

```diff
@@ -11,10 +17,20 @@ const authRouter = require('./routes/auth');
 const notFoundMiddleware = require('./middleware/not-found');
 const errorHandlerMiddleware = require('./middleware/error-handler');
 
+app.set('trust proxy', 1);
+app.use(
+  rateLimit({
+    windowMs: 15 * 60 * 100,
```

Comment:
Just double checking if this is what you intended. This ends up with 90,000ms which is 90 seconds or 1 and a half minutes. I believe in the video the instructor sets this to 15 minutes which would be: `15 minutes * 60 seconds in a minute * 1000 milliseconds in a second` ie: `windowMs: 15 * 60 * 1000`, or 900,000 milliseconds.